automation-tests/lib/wd-extensions.js

```
@@ -153,7 +161,7 @@ wd.prototype.wwin = function(opts, cb) {
     cb = arguments[0];
     self.windowHandles(function(err, handles) {
       if (err) return cb(err);
-      self.window(handles[0], function(err) { cb(err); }); // fire cb whether err is defined or not
+      self.window(browser._parentWindow, function(err) { cb(err); }); // fire cb whether err is defined or not
```

browser does not exist in this context, _parentWindow is on self at this point. This is causing a silent failure where all of the tests appear to pass but where all the tests continue running on sauce.